### DISCUSSION TITLE
New to this..

### DISCUSSION THREAD

**Discussion by pestiforousalt** [thatskymod/Sky-CotL-Scripts#101] [Q&A]
Greetings to all, I literally made an account so I could try to get either answers or help with this.

I'm definitely a relatively new player to Sky. I enjoy the social aspect, but the grindy stuff just gets so boring for me, as well as exhausting. And with having just turned 21, lifes gonna get a lot more hectic with my job and having to obtain things like insurance, looking into a place to rent, etc.

I just _don't_ have time to grind this game for cosmetics anymore I won't wear more than once or even a few times. The bugs, the weird layout with friending and doing candle runs, it's all very overwhelming. But with how much the devs BOAST about being a game about creating relationships and being creative, they sure do limit it with IAPs and their season passes. It's not that I'm lazy either. Genuinely taking hours upon hours bumping into every damn bug on the way for merely 20 candles for one CR before reset is truly exhausting. I applaud those who commit. I can't do that now.

But, the issue is that I started on PC. I have no idea how I'd apply these mods and whatnot this way, considering many of these mods and whatnot are for everything else. And I don't wish to put my account at risk. I read somewhere that things like Canvas modding won't get an account flagged as long as one isn't waving around cheats, but you never know.

If anyone could give insight or explain this whole modding scene, that'd be awesome. I hate to be a bother or sound like it, but I'm kinda fed up with how greedy TGC is being just for cosmetics.

**Comment by BubblePOPVA** [comment]
Some buttons of mine wont work and I have no clue as of why.
